### DISCUSSION TITLE
Raimadim WiFi AC1300 [RTL8822BU]

### DISCUSSION THREAD

**Discussion by comrade-meowski** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#343] [Show and tell]
Pleased to report this el cheapo dongle working perfectly with Wireless USB OC Big Sur Adapter-V17. 

It's confusingly named and variously identifies itself as a Raimadim AC1300 and/or Techkey AC1200 but uses the RTL8822BU chipset. Linux loads the rtw_8822bu module for it. The unit works fine on windows too so for 10 quid it's a handy universal usb wifi dongle.

It's this one:  https://www.amazon.co.uk/Raimadim-WiFi-Dongle-AC1300Mbps-Wireless-black/dp/B0CLCXRX2S

Tested on a MacBookPro11,3 + OCLP 1.5.0 + macOS 14.5.

Thanks for the software @chris1111!

**Comment by chris1111** [comment]
Thanks @comrade-meowski 
 I will add to the support list
